plugins/com.google.cloud.tools.eclipse.appengine.deploy/src/com/google/cloud/tools/eclipse/appengine/deploy/standard/StandardDeployJob.java

```diff
@@ -86,7 +85,8 @@ private CloudSdk getCloudSdk() {
     MessageConsole messageConsole =
         MessageConsoleUtilities.getMessageConsole(CONSOLE_NAME, null, true /* show */);
     final MessageConsoleStream outputStream = messageConsole.newMessageStream();
-    CloudSdk cloudSdk = new CloudSdkProvider().createBuilder()
+    CloudSdk cloudSdk =
```

Comment:
@akerekes: I was wondering if the CloudSdk instance shouldn't be configured outside of the job, via `CloudSdkPrompter`, as it's possible someone imported a project and so has not yet configured the SDK.  Plus the DeployJob may be used in a background or headless fashion, so the message-console hookups may not be desired.